flatnav/util/Reordering.h

```diff
@@ -137,7 +137,7 @@ rcmOrder(std::vector<std::vector<node_id_t>> &outdegree_table) {
          const std::pair<node_id_t, int> &b) { return a.second < b.second; });
 
   std::vector<node_id_t> P;
-  ExplicitSet is_listed = ExplicitSet(cur_num_nodes);
+  VisitedNodesHandler is_listed = VisitedNodesHandler(cur_num_nodes);
```

Comment:
Nit: Not sure about the name - maybe VisitedNodeTracker or VisitedNodeSet instead of VisitedNodesHandler? (handler could do any number of things, including pruning, sort-by-distance, etc).